### DISCUSSION TITLE
IPCスタブの自動生成ツールが間違えている可能性がある

### DISCUSSION THREAD

**Discussion by ohfujimasa73** [nuta/microkernel-book#7] [Bugs]
### 概要

tool/generate_ipcstub.pyのリプライ(応答)がメッセージ送信と同じ構造体を指定しているが意図しているものか

![image](https://github.com/nuta/microkernel-book/assets/26175410/a7f65e81-9095-4769-a4b1-e945719ad15c)

下記のように"reply_"が必要と思われる。
ついていないためメッセージ送信、応答が同じ構造体、データになっている

![image](https://github.com/nuta/microkernel-book/assets/26175410/92e7b221-333e-4804-bb3b-918cebb29972)

### 開発環境

```bash
ブランチ:main,fix-UnicodeDecodeError-in-windows
```

**Comment by nuta** [comment]
ご報告ありがとうございます。確かに`reply_`が欠けていますね。現状使われていないフィールドなので、削除する方向で修正しようと思います 😃

**Reply by nuta** [reply]
修正PR: https://github.com/nuta/microkernel-book/pull/8
